x-pack/test/functional/es_archives/event_log_multiple_indicies/mappings.json

```diff
@@ -0,0 +1,2685 @@
+{
```

Comment:
yikes!  you'd think there would be something _smaller_ you could do, heh.  

Is this needed because the test is using the `event_log_test_ saved object fixture type?  We _could_ probably change that, or create a different test, that used alerts instead, if it would avoid having to pull in this massive wad.

Reply:
Yeah, it's a pain... esArchiver isn't the esasiest to debug when it just decides not to load the archive anymoe so left it as is.

I'e now pruned a bunch of the mappings... 🤞

Reply:
Not a big deal, wouldn't spend a lot of time on it.  This will always just be testing the 7.9 -> 8.0 cross-over, so it doesn't seem like it will ever need to change in the future.